packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.tsx

```diff
@@ -197,6 +212,50 @@ class HorizontalOverflow extends Foundation<
                 itemsHeight,
             });
         }
+
+        if (this.props.onHorizontalOverflowChange) {
+            this.onHorizontalOverflowChange();
+        }
+    };
+
+    /**
+     * Handles the resize event
+     */
+    private onWindowResize = (): void => {
+        this.onHorizontalOverflowChange();
+    };
+
+    /**
+     * Checks if overflow is occuring
+     */
+    private checkOverflow(): boolean {
+        const regionWidthInformation: any = this.getItemsWidthAndTotalWidth();
+        const availableWidth: number = regionWidthInformation.availableWidth;
+        const itemWidths: number[] = regionWidthInformation.itemWidths;
+        const totalItemWidth: number = itemWidths.reduce((a: number, b: number) => a + b);
+
+        return totalItemWidth < availableWidth ? false : true;
```

Comment:
@nalogiudice this statement should return bool:

```
return totalItemWidth < availableWidth;
```

Reply:
Duh yeah.